src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/PaneLayoutPreferencesPane.java

```diff
@@ -145,6 +145,13 @@ public void setValue(ArrayList<String> tabs)
             checkBox.setValue(tabs.contains(checkBox.getText()), false);
       }
 
+      public boolean presentationVisible()
+      {
+         CheckBox lastCheckBox = checkBoxes_.get(checkBoxes_.size() - 1);
```

Comment:
Is there any risk of this being called where checkBoxes_ is empty, so checkBoxes_.size() - 1 would be negative and (I think) end up throwing an exception? Might be worth a check for this if any chance of it happening.

Reply:
There's a slight chance this could happen if called on an empty TabSet2. Added a check, thanks :)